troubleshooting/README.md

```diff
@@ -0,0 +1,63 @@
+# Troubleshooting the Environment
+Kubernetes cluster configuration could be complex. To run the app with `Application Insights for Kubernetes` on does not always work for the first shoot. It is a challenge when it is not clear where the issue comes from - the configuration for Kubernetes, the build of the images or the library of `Application Insights for Kubernetes`.
```

Comment:
.. does not always work on the first try

Reply:
Will fix this. Thanks!